ui/src/app/shared/components/first-time-user-panel.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import {useEffect, useState} from 'react';
+
+// This wraps the children components in a FTU which allows you to provide an explanation of how to use the feature.
+// The child components are hidden behind the panel until the users dismissed the panel by clicking "Continue".
+// The fact the users dismissed the panel is persisted in local storage and the panel not show again.
+export const FirstTimeUserPanel = ({id, explanation, children}: {id: string; explanation: string; children: React.ReactElement}) => {
```

Comment:
Perhaps move this to a separate PR?

Reply:
Kinda of a small PR. Do you mean before or afterwards?

Reply:
It's up to you. It's just for narrowing down the changes in this PR to make it easier for others to review but I don't have a strong opinion on moving them to a different PR.